README.md

```diff
@@ -54,6 +54,12 @@ The following reference implementations are provided:
   * aces_1.0.2
   * aces_1.0.3
 
+The [Colour Science](https://github.com/colour-science/OpenColorIO-Configs) repository maintained by Thomas Mansencal contains the following additional configurations:
```

Comment:
`[Colour Science]` -> `[Colour Science OpenColorIO-Configs]`

`repository` -> `fork`

Reply:
Just for clarity

Reply:
Changes made, just need to push.